### PR TITLE
Zongdaoming/writingpapers

### PR DESCRIPTION
**Pull request by zongdaoming**
try it ...

### CODE REVIEW

File: source/_posts/how-to-writing-paper.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
+---
+layout: blog
+title: Hwo to write paper
```
Comment (Baileyswu) [inline source/_posts/how-to-writing-paper.md]: Hwo -> How

File: source/_posts/how-to-writing-paper.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+## 段落的写法
+
+>* 每个段落有个论断性的<font color='red'>中心句</font>
+>* 其余部分都是<font color='blue'>支撑句</font>，围绕中心句展开论证
+   >>* 前人工作
+   >>* 具体数据
+>* 支撑句之间可分类组织,支撑句要论证严密
+>* 段尾可以加上<font color='purple'>衔接句</font>
+### 图和表的重要性
+>* 图和表是论文的骨架，争取<font color='green'>让读者按照顺序看就能理解论文的主要思想，不用通过看正文才能懂</font>
```
Comment (Baileyswu) [inline source/_posts/how-to-writing-paper.md]: remove >

File: source/_posts/how-to-writing-paper.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+author: Zong Daoming
+mathjax: true
+summary: 论文写作
+categories: paper writing
+tags:
+  - writing paper
+  - logic expression
+---
+
+# How to write your first SCI paper?
```
Comment (Baileyswu) [inline source/_posts/how-to-writing-paper.md]: move this main title to line 3

Comment (zongdaoming) [reply source/_posts/how-to-writing-paper.md]: Fixed already

File: source/_posts/how-to-writing-paper.md (hunk 2 above)
Comment (zongdaoming) [reply source/_posts/how-to-writing-paper.md]: already fixed it

File: source/_posts/docker-usage.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+date: 2019-09-12 04:29:08
+author: zongdaoming
+summary: Docer的入门级使用
+categories: tools
+---
+# Docker从入门到放弃
+----
+> 作者: [宗道明](https://github.com/zongdaoming)
+>
+> 校验: [吴丽丹](https://github.com/Baileyswu)
```
Comment (Baileyswu) [inline source/_posts/docker-usage.md]: remove checker

**Pull request merged**